config/interface/hub/user_tasks.yml

```diff
@@ -868,19 +868,19 @@ debug_geographic_item_task:
   description: 'Report metadat about a geographic item.'
 new_lead_task:
   hub: true
-  name: 'New dichotomous key'
+  name: 'New multifurcating key'
```

Comment:
See above, I think we can just do "New key", and "New interactive key", and folks will get familiar with that pattern.